processor/transformprocessor/internal/common/functions.go

```diff
@@ -117,6 +148,11 @@ func NewFunctionCall(inv Invocation, functions map[string]interface{}, pathParse
 				}
 				args = append(args, reflect.ValueOf(arg))
 				continue
+			case "int64":
+				if argDef.Int == nil {
+					return nil, fmt.Errorf("invalid argument at position %v, must be an int", i)
+				}
+				args = append(args, reflect.ValueOf(*argDef.Int))
```

Comment:
this change seem unrelated to the truncate functionality

Reply:
This addition allows the new  truncate_all function to be called with a second parameter that is an int.  Without this, the `args` variable won't contain the second parameter that is an int64.

Reply:
My bad, i was confused by the spacing in the github diff 👍